app/Filament/Components/BreadcrumbPreview.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Filament\Components;
+
+use App\Models\GameSet;
+use App\Platform\Actions\BuildHubBreadcrumbsAction;
+use Filament\Infolists\Components\Entry;
+use Illuminate\Support\HtmlString;
+
+class BreadcrumbPreview extends Entry
+{
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        $this->view('filament.components.breadcrumb-preview');
+    }
+
+    public function getValue(): ?HtmlString
+    {
+        /** @var GameSet $gameSet */
+        $gameSet = $this->getRecord();
+
+        if (!$gameSet || !($gameSet instanceof GameSet)) {
+            return null;
+        }
+
+        $breadcrumbs = (new BuildHubBreadcrumbsAction())->execute($gameSet);
+
+        if (empty($breadcrumbs)) {
+            return null;
+        }
+
+        /**
+         * Check if the hub is orphaned. We consider a hub potentially orphaned if:
+         *  1. The breadcrumb path has 2 (or fewer) entries. Typically this means it's just
+         *     [Central] -> [My Hub] with no intermediate parents. A properly connected hub
+         *     should usually have at least 3 entries like [Central] -> [Central - Type] -> [Type - Hub].
```

Comment:
I assume this will be fixed by #2947 
![image](https://github.com/user-attachments/assets/5398cb2a-03f5-432a-a572-6f68af6cd2b9)

Reply:
It is indeed (which is now merged into this branch).

<img width="851" alt="Screenshot 2024-12-19 at 9 09 40 PM" src="https://github.com/user-attachments/assets/ac5cb5c4-4bc0-4a7d-b4ab-0a4ac3296307" />

As a future feature, we may also want to consider if it's desirable for someone managing the hub to simply "force" what the parent hub is. I haven't thought fully through how such functionality might be built, but it could be desirable.